lib/sycamore/sycamore/transforms/text_extraction/ocr_models.py

```diff
@@ -173,67 +175,41 @@ class PaddleOcr(OcrModel):
     # NOTE: Also requires the installation of paddlepaddle or paddlepaddle-gpu
     # depending on your system
     @requires_modules(["paddleocr", "paddle"], extra="local-inference")
-    def __init__(self, language="en", slice_kwargs={}, **kwargs):
-        from paddleocr import PaddleOCR
-        from paddleocr.ppocr.utils.logging import get_logger
+    def __init__(self, language="en", **kwargs):
```

Comment:
This needs tests. I suggest two tests:
1. Run paddle on some interesting content, capture it and replace self.reader with something that returns the captured content. Then have a unit test that verifies we transform paddles output properly. 
2. Run an integration test on 1-2 pages where 3.0 works better than 2.0. Verify that the OCR gets the better results. This is an integration test because it will be slower, but this test will check that things work if they update the interface unexpectedly.